index.js

```diff
@@ -4,10 +4,51 @@ const mw = require("./lib/middleware");
 const actions = require("./lib/actions");
 const app = require("express")();
 const bodyParser = require("body-parser");
+const morgan = require("morgan");
+const winston = require("winston");
 const argv = require("minimist")(process.argv.slice(2));
 const bodyLimit = argv.limit || "50mb";
 const port = argv.port || 3000;
 
+process.on("SIGINT", () => {
+  console.info("Interrupted");
+  process.exit(0);
+});
+
+const logger = winston.createLogger({
+  transports: [
+    new winston.transports.Console({
+      level: "info",
+      json: true,
+    }),
+  ],
+});
+
+const morganJSONFormat = () =>
+  JSON.stringify({
+    method: ":method",
+    url: ":url",
+    http_version: ":http-version",
+    remote_addr: ":remote-addr",
+    remote_addr_forwarded: ":req[x-forwarded-for]", //Get a specific header
+    response_time: ":response-time",
+    status: ":status",
+    content_length: ":res[content-length]",
+    timestamp: ":date[iso]",
+    user_agent: ":user-agent",
+  });
```

Comment:
Judging by the [morgan docs](https://www.npmjs.com/package/morgan#using-format-string-of-predefined-tokens) I'm surprised this works?

I'd have thought that
```
[
    ":method",
    ":url",
    ":http-version",
    ":remote-addr",
    ":req[x-forwarded-for]", //Get a specific header
    ":response-time",
    ":status",
    ":res[content-length]",
    ":date[iso]",
    ":user-agent",
].join(" ")
```
would produce a "format string of predefined tokens" as per the docs.